ISSUE-804 - Add support for custom labels in Ingress

rob8714 opened the pull request:
PR for #804

kishorv10 commented:
@nikhilparmar86 Can you review the constellation chart changes?

rob8714 commented:
@kishorv10 @nikhilparmar86 wanted to check if there's an ETA when this could be reviewed and merged?

kishorv10 commented on charts/pega/values.yaml:
```diff
@@ -220,6 +220,8 @@ global:
         # path:
         # pathType:
+        labels:
+        # Configure custom labels for ingress
```
probably comment is misplaced here?

kishorv10 commented:
@rob8714 Can you add few terraform tests for these changes? [Here](https://github.com/pegasystems/pega-helm-charts/tree/master/terratest/src/test/pega) are the existing tests.

rob8714 commented:
@kishorv10 I have fixed the issue with the comment and added one test.  I am not a developer and not really familiar with go, so got some help to write the test from one of the online Gen AI tools.  I ran the test and it looks to be working fine, if it's ok and you would like more tests added let me know.
